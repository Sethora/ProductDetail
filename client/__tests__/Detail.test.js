import React from 'react';
import { mount } from 'enzyme';
import Detail from '../src/components/detail';

describe('Detail component', () => {
  it('renders', () => {
    const component = mount(<Detail title="Noe you see me" text="Now you don't" />);
    expect(component).toMatchSnapshot();
  });
});