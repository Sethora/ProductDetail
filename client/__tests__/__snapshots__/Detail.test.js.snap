// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Detail component renders 1`] = `
<Detail
  text="Now you don't"
  title="Noe you see me"
>
  <styled.div>
    <div
      className="sc-bdfBwQ kqAbSP"
    >
      <p
        className="fan"
      >
        <span
          data-testid="spantag"
        >
          Noe you see me
           
        </span>
        Now you don't
      </p>
    </div>
  </styled.div>
</Detail>
`;
